project/plugins.sbt

```diff
@@ -1,10 +1,12 @@
-addSbtPlugin("com.eed3si9n"               %   "sbt-unidoc"              % "0.4.1")
-addSbtPlugin("com.github.gseitz"          %   "sbt-release"             % "1.0.6")
-addSbtPlugin("com.jsuereth"               %   "sbt-pgp"                 % "1.1.0")
-addSbtPlugin("org.scoverage"              %   "sbt-scoverage"           % "1.5.1")
-addSbtPlugin("com.timushev.sbt"           %   "sbt-updates"             % "0.3.1")
-addSbtPlugin("com.typesafe.sbt"           %   "sbt-ghpages"             % "0.6.2")
-addSbtPlugin("com.updateimpact"           %   "updateimpact-sbt-plugin" % "2.1.3")
-addSbtPlugin("org.xerial.sbt"             %   "sbt-sonatype"            % "2.0")
-addSbtPlugin("com.codacy"                 %   "sbt-codacy-coverage"     % "1.3.11")
-addSbtPlugin("io.github.davidgregory084"  %   "sbt-tpolecat"            % "0.1.3")
+addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.1")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.6")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
```

Comment:
Not a big deal but I liked the previous alignment :)

Reply:
Yeah, I did like the previous alignment! I will try to find a way to keep original alignment.